manifests/broker.pp

```diff
@@ -0,0 +1,20 @@
+# Set up the broker
+class pulp::broker {
+  if $pulp::manage_broker {
```

Comment:
Since I don't have a true feel for this, I try to look at what other well defined modules do. For example, the puppetdb module from Puppetlabs (https://github.com/puppetlabs/puppetlabs-puppetdb) follows this type of pattern, where, because it requires a Postgres server to run will set one up unless a parameter is specified to indicate that the database shouldn't be managed by the module. I think that is where I tend to try to draw my line -- what is the minimum required to have a functioning Pulp and the module should set that up by default.